.github/workflows/release-embedding-sdk.yml

```diff
@@ -245,14 +245,15 @@ jobs:
           path: sdk
 
       - name: Publish to NPM
+        working-directory: sdk
         run: |
-          cd sdk
           echo //registry.npmjs.org/:_authToken=${{ secrets.NPM_RELEASE_TOKEN }} > .npmrc
           # Please keep the value in sync with `inputs.branch`'s release branch
           npm publish --tag ${{fromJson('{"master": "canary", "release-x.51.x": "51-stable"}')[inputs.branch]}}
 
       - name: Add `latest` tag to the latest release branch (`release-x.51.x`) deployment
         if: ${{ inputs.branch == 'release-x.51.x' }}
+        working-directory: sdk
```

Comment:
The problem was that `.npmrc` was created inside the `sdk` directory, so we need to be in the same directory as the last step otherwise, we'll get [an authentication error](https://github.com/metabase/metabase/actions/runs/11498053683/job/32003556362#step:15:8). I use `working-directory` to make it clear instead of using `cd sdk`.

We're now in the `sdk` directory after doing this. Here's the [test result](https://github.com/metabase/metabase/actions/runs/11498415381/job/32004651830#step:14:12), you can see that we can see all the `dist` folder which will be the files we published to npm.